scripts/postinstall.js

```diff
@@ -4,9 +4,7 @@ const chalk = require('chalk');
 // git v2.9.0 supports a custom hooks directory. This means we just need to checkin the hooks scripts
 spawnSync('git', ['config', 'core.hooksPath', '.githooks']);
 
-console.log(`${chalk.green(
-  'All dependencies are installed! This repo no longer builds automatically after installing dependencies.'
-)}
+console.log(`${chalk.green('All dependencies are installed! This repo no longer builds automatically after installing dependencies.')}
```

Comment:
No idea why this change happened... doesn't make any sense

Reply:
Probably because prettier was run while merging the commit from master that contained this change. (But it also should have run when this was committed in the first place, so not sure why it would be formatted differently this time.)